package/MDAnalysis/lib/formats/include/readdcd.h

```diff
@@ -136,7 +136,7 @@ static int write_dcdheader(fio_fd fd, const char *remarks, int natoms,
  * Output: 0 on success, negative error code on failure.
  * Side effects: coordinates are written to the dcd file.
  */
-static int write_dcdstep(fio_fd fd, int curstep, int curframe, 
```

Comment:
> My other assumption was that long-term we'd replace the plugin C code, and maybe even try to have something that becomes progressively faster--part of the objective to have independent format readers, etc. Maybe after asv or some benchmarking library is used to make sure our performances are improving over the project lifespan.

It's hard to beat the VMD molfile plugin code on speed; a lot of experience has gone into writing these. Also, why re-invent the wheel if there's a open and free solution out there? Ideally we would solve #55 (have a nice open source python binding for the VMD mofile plugin that compiles everything easily).

One reason for working on our own is to support certain formats in better ways. For instance, VMD takes a NAMD-centric view towards DCDs as far as I can tell.

>> why isn't the C just handling that for the format it is supposed to support
> DCD is old and there doesn't exist a proper standard.

There's one issue thread (probably on unit cells) that links to recent NAMD specs, and some of the mdtraj folks also had comments (or mentioned one of our issues). Perhaps it would be worthwhile to create a wiki page on problematic formats and gather what we know.